Add Meeting Artifacts 5 Sept 2024 (#256)
confidential-computing/governance: TAC/Meetings/2024/2024-09-05/CCC-TAC-Minutes-2024-09-05.md

* **CCC YouTube**: [youtube.confidentialcomputing.io](https://youtube.confidentialcomputing.io)
* **LFX**: [lfx.linuxfoundation.org](https://lfx.linuxfoundation.org)
* **Join the CCC**: [join.confidentialcomputing.io](https://join.confidentialcomputing.io)
* **Contact the CCC**: [confidentialcomputing.io/contact-us/](https://confidentialcomputing.io/contact-us/)

### Voting members of the TAC

*Per the [charter](https://charter.confidentialcomputing.io), all [CCC Premier members](https://confidentialcomputing.io/members/) receive one vote on the TAC. Quorum for votes is at least 50% of voting members present.*

* AMD - David Kaplan + / Harold Gilkey
* Arm -  Michael Lu
* Google - Catherine Zhang +
* Huawei - Zhipeng (Howard) Huang
* Intel - Dan Middleton (TAC chair) + / Simon Johnson
* Meta -  Henry Wang + /  Kevin Hui
* Microsoft - Alec Fernandez +
* Nvidia - Fritz Alder
* Red Hat - Yash Mankad+  / Ram Pai
* TikTok - Mingshen Sun +  / Yao Zhang

   " + " indicates attending

#### Other Attendees

* Alexander Manilov
* Andey Pogoreltev (Super Protocol)
* Chandra Nelogal (Self)

> * Diona Glaze

Reviewer: ```suggestion
* Diona Glaze (Google)
```

* Henk Birkholz
* Hesham (Innovax Technologies)
* Jens Albers (Fr0ntierX)
* Kanth Ghatraju (Oracle)
* Ken Fromm (DSA)
* Leonardo Garcia (Linaro)
* Manu Foutaine (Hushmesh)
* Mark Novak (JP Morgan Chase)
* Nukri Super (Super Protcol)
* Nathaniel McCallum (AMD)
* Ofir Azoulay-Rozanes (Anjuna)
* Rahel Wan (IBM)
* Ram Pai (IBM)
* Simon Frost (ARM)
* Stephen Smalley (NSA)
* Vladimir Haltakov (Fr0nrierx)
* Yulia Gontar (Super Protocol)
* Mike Bursell (CCC)
* Sal Kimmich (CCC)
* Renu Chauhan (LF)
* Riaan Kleinhans (LF)

## Welcome

Dan Middleton (DM) opened the call at 7:04 am PT and reviewed the antitrust policy. 

## Agenda

DM reviewed the agenda for the meeting.

## Roll Call / Attendance

> Quorum was established as no votes were required.

Reviewer: ```suggestion
Quorum was established although no votes were required.
```

## Attestation Terminology

Mike Bursell (MB) introduced Attestation & AVS (Attestation Verification Services) and discussed the topics that came out of the Attestations Business models document. He discusses the way CCC should talk about attestation and remote attestation. He proposed that definitions need to be calcified. Chandra Nelogal (CN) agreed that the term remote could mean several different things and continued to explain his perspective on the topic. Nathaniel McCallum (NM) said that Attestation is becoming a loaded term with different definitions in various industries. Further discussion ensued.

DM suggested that the work should be broken down into work packages that could be addressed through working groups. Catherine Zhang ask if the work packages would be addressed by existing working groups or new groups. MB proposed that a portion of the TAC meeting could be set apart for that on a bi-weekly basis. DM proposed that the TAC could assign questions and work to the existing working groups to be addressed. DM asked meeting participants to communicate on the topic through the TAC Slack channel and mailing list and it would be further discussed in the next meeting.

> ## Super Protoco Tech Talk

Reviewer: ```suggestion
## Super Protocol Tech Talk
```

> Nukri Basharuli (Founder/CEO) introduced themselves and the Super Protocol team. Nukri and Andrey Pogoreltsev - CTO of Super Protocol continued to present their tech talk. [Presentation](./CCC_Super%20Protocol%20Tech%20overview_Sept%205'24.pdf)

Reviewer: update link

Author: Good catch. Done.

##  2024 CCC TAC Objectives

> DM introduced the TAC goals document. Yash Mankad updated the meeting on the progress on the Mentorship program and thanked SK for driving participation from the CCC projects. They anticipate receiving two project proposals soon from the Veriosin project and are discussing with Coconut SVSM. Sal Kimmich (SK) said the Veriason would be mentoring two mentees. Riaan would help determine the administrative requirements of one mentor working with two mentees.

Reviewer: ```suggestion
DM introduced the TAC goals document. Yash Mankad updated the meeting on the progress on the Mentorship program and thanked Sal Kimmich (SK) for driving participation from the CCC projects. They anticipate receiving two project proposals soon from the Veraison project and are discussing with Coconut SVSM. SK said the Veraison would be mentoring two mentees. Riaan will help determine the administrative requirements of one mentor working with two mentees.
```
